sass/theme/_theme.scss

```diff
@@ -112,3 +112,20 @@ position:relative;
 .acf-block-preview .card-button a{
 	pointer-events: none;
 }
+
+/*--------------------------------------------------------------
+Override UDS scss
+--------------------------------------------------------------*/
+
+//Override the UDS hero class on heroes.scss
+@media screen and (max-width: 576px){
```

Comment:
I think we should probably use the variables from `_variables.scss`, rather than actual pixel dimensions, on the small chance that someone changes the breakpoints. I think using `$uds-breakpoint-sm` will give us the same 576px, but also update itself if the small breakpoint is ever changed.